elytron/src/test/java/org/wildfly/extension/elytron/SyslogAuditLogTestCase.java

```diff
@@ -0,0 +1,327 @@
+     * by verifying the connection message
+     */
+    @Test
+    public void testRfc5424SyslogFormat() throws Exception {
+        udpOperation.get(ElytronDescriptionConstants.SYSLOG_FORMAT).set(RFC5424_STRING);
+        assertSuccess(services.executeOperation(udpOperation));
+        verifyUdpMessage(RFC5424_SYSLOG_MESSAGE);
+    }
+
+
+    /**
+     * Doesn't set a syslog and verifies the Elytron AuditEndpoint is created with the default value
+     * of RFC5424 by verifying the connection message
+     */
+    @Test
+    public void testDefaultSyslogFormat() throws Exception {
+        assertSuccess(services.executeOperation(udpOperation));
+        verifyUdpMessage(RFC5424_SYSLOG_MESSAGE);
+    }
+
+    /**
+     * Tests that the syslog configuration operation fails if an invalid RFC format is used
+     */
+    @Test
+    public void testWrongSyslogFormat() {
+        udpOperation.get(ElytronDescriptionConstants.SYSLOG_FORMAT).set(BAD_RFC_STRING);
+        ModelNode response = services.executeOperation(udpOperation);
+        assertCorrectError(response, "No enum constant org.jboss.logmanager.handlers.SyslogHandler.SyslogType.RFC1");
+        assertFailed(response);
+    }
+
+    /**
+     * Tests that the syslog configuration operation fails if an invalid icmp-error-reconnect-attempts number is used
+     */
+    @Test
+    public void testBadIntegerReconnectAttempts() {
+        udpOperation.get(ElytronDescriptionConstants.ICMP_ERROR_RECONNECT_ATTEMPTS).set(BAD_RECONNECT_NUMBER);
+        ModelNode response = services.executeOperation(udpOperation);
+        assertCorrectError(response, BAD_RECONNECT_NUMBER + " is an invalid value for parameter icmp-error-reconnect-attempts. A minimum value of -1 is required");
+        assertFailed(response);
+    }
+
+    /**
+     * Tests that the server does not attempt to resend the message
+     */
+    @Test
+    public void testZeroReconnectAttemptsBadHost() {
+        // Windows Server can have an issue echoing back an error, causing the test to fail by not seeing a failure sending
+        Assume.assumeTrue("Test does not run on Windows Server", SystemUtils.OS_NAME.contains("Windows Server"));
+        badHostUdpOperation.get(ElytronDescriptionConstants.ICMP_ERROR_RECONNECT_ATTEMPTS).set(0);
+        ModelNode response = services.executeOperation(badHostUdpOperation);
+        assertCorrectError(response, "The maximum icmp-error-reconnect-attempts value of 0 was reached");
+        assertFailed(response);
+    }
+
+    /**
+     * Tests that the server reattempts to send the message {@link SyslogAuditLogTestCase#RECONNECT_NUMBER} times
+     */
+    @Test
+    public void testNumberedReconnectAttemptsBadHost() {
+        // Windows Server can have an issue echoing back an error, causing the test to fail by not seeing a failure sending
+        Assume.assumeTrue("Test does not run on Windows Server", !SystemUtils.OS_NAME.contains("Windows Server"));
```

Comment:
This one seems to be the opposite of above, but the same comments. I'm guessing this should be:
```
Assume.assumeFalse("Test does not run on Windows Server", SystemUtils.OS_NAME.contains("Windows Server"));
```

Reply:
Updated